pkg/operators/wasm/syscalls.go

```diff
@@ -28,8 +28,11 @@ import (
 
 func (i *wasmOperatorInstance) addSyscallsDeclarationsFuncs(env wazero.HostModuleBuilder) {
 	exportFunction(env, "getSyscallName", i.getSyscallName,
-		[]wapi.ValueType{wapi.ValueTypeI32}, // Syscall ID
-		[]wapi.ValueType{wapi.ValueTypeI64}, // Syscall Name
+		[]wapi.ValueType{
+			wapi.ValueTypeI32, // Syscall ID
+			wapi.ValueTypeI64, // Buffer to save syscall name
+		},
+		[]wapi.ValueType{wapi.ValueTypeI64}, // Error
```

Comment:
Why not return value of `ValueTypeI32`? Other functions that return "0 in case of success, 1 otherwise" are `ValueTypeI32` so it is not consistent.

Reply:
My bad. It was used to return the name too, so I only changed the description without checking the type. Thanks for catching this up!